app/models/preserved_copy.rb

```diff
@@ -40,6 +40,10 @@ class PreservedCopy < ApplicationRecord
     joins(:endpoint).where(endpoints: { storage_location: storage_dir })
   }
 
+  scope :by_druid, lambda { |druid|
+    joins(:preserved_object).where(preserved_objects: { druid: druid })
+  }
+
```

Comment:
Should there be tests for this scope?  Things like:

- no copies
- one copy
- multiple copies

Should this scope be used by CV check?